### PR TITLE
build(): build a -rtl.css file along the css, and compile _every_ sass files

### PR DESCRIPTION
**Pull request by hansl**
It's simpler.

### CODE REVIEW

File: ember-cli-build.js [hunk 1]
```diff
@@ -21,27 +23,52 @@ module.exports = function(defaults) {
 };
 
 
+/** Generate RTL CSS files along side regular CSS. */
+class BroccoliSassWithRtl extends BroccoliSass {
```
Comment (jelbourn) [inline ember-cli-build.js]: Would it be better to make a standalone broccoli-cssjanus plugin (similar to how autoprefixer is implemented)?

File: ember-cli-build.js [hunk 2]
```diff
@@ -21,27 +23,52 @@ module.exports = function(defaults) {
...
+    const cssRtlOutputPath = path.join(this.outputPath,
+        path.dirname(this.outputFile),
+        path.basename(this.outputFile, '.css') + '-rtl.css');
+    fs.writeFileSync(cssRtlOutputPath, cssjanus.transform(fs.readFileSync(cssOutputPath).toString()));
+  }
+}
+
+
+
 /** Gets the tree for all of the components' CSS. */
```
Comment (jelbourn) [inline ember-cli-build.js]: Update function comment

File: ember-cli-build.js [hunk 3]
```diff
@@ -21,27 +23,52 @@ module.exports = function(defaults) {
...
-      return BroccoliSass(
-        [`src/components/${component}`, 'src/core/style'], // Directories w/ scss sources
-        `./${fileName}.scss`,                              // Root scss input file
-        `components/${component}/${fileName}.css`);        // Css output file
-    }).concat(trees);
-  }, []);
-
-  return broccoliAutoprefixer(mergeTrees(componentCssTrees), autoprefixerOptions);
+function getCssTree(source, destination) {
+  function walk(dir) {
```
Comment (jelbourn) [inline ember-cli-build.js]: Move the `walk` function out of the function body of `getCssTree` and add a description.

**Comment by hansl** [comment]
This will not make it in, as we decided that using RTL specific handmade CSS was better (for the time being). We can always revive this PR when needed.

**Pull request closed without merging**